prov/verbs/src/verbs_domain.c

```diff
@@ -200,6 +229,14 @@ static struct fi_ops fi_ibv_fid_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static struct fi_ops fi_ibv_rdm_fid_ops = {
+	.size = sizeof(struct fi_ops),
+	.close = fi_ibv_domain_close,
+	.bind = fi_ibv_rdm_domain_bind,
```

Comment:
manipulating by ops pointers may confuse developers (it is not clear if some API is changed after initialization).
FI_MR_COMPLETED will be added to EP_MSG later, after this we will merge these ops